app/models/ability.rb

```diff
@@ -42,10 +42,11 @@ def initialize(user)
     if user.role? :tutor
```

Comment:
Maybe that's also a lack of communication between teams. Whit us, the word "tutor" was never even mentioned.